### DISCUSSION TITLE
UnsafeAccessorAttribute with properties

### DISCUSSION THREAD

**Discussion by RikkiGibson** [dotnet/runtime#97417] [Q&A] [answered]
re: #81741

It looks like there isn't an explicit mention of properties here. Is it expected/supported to use `[UnsafeAccessor]` to call property accessors, e.g. using the property accessor's metadata name, and matching the metadata signature of the accessor method?

cc @AaronRobinsonMSFT

**Comment by AaronRobinsonMSFT** [comment, marked as answer]
@RikkiGibson Yes, properties are supported and work as one would imagine. See the following test for calling a the getter and setter on a property.

https://github.com/dotnet/runtime/blob/6f24dbcfccde55accd5a9fa0a86f1f4cf8fa1075/src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs#L377C1-L382C73

**Reply by RikkiGibson** [reply]
Thanks! It might be reasonable to update the documentation to show this.

**Reply by AaronRobinsonMSFT** [reply]
Done https://github.com/dotnet/dotnet-api-docs/pull/9569
